lib/dockerregistry/inventory.go

```diff
@@ -69,60 +70,6 @@ func MakeSyncContext(
 		RegistryContexts:    rcs}, nil
 }
 
-// Basic logging.
-
-// Infof logs at INFO level.
-func (sc *SyncContext) Infof(s string, v ...interface{}) {
-	if sc.Verbosity > 2 {
```

Comment:
The old system assigned the value of verbosityPtr to sc.Verbosity, like this: https://github.com/kubernetes-sigs/k8s-container-image-promoter/blob/cca667f4e15627f654379a7a4ceacaa83d89d570/cip.go#L130. Can you somehow make our new use of klog observe the intent of verbosityPtr as well?

Reply:
Would something like the following be sufficient? I can't think of a better way to implement it

Instread of:
```
if *garbageCollectPtr {
    klog.Infof("---------- BEGIN GARBAGE COLLECTION: %s ----------\n",
        *manifestPtr)
...
```
Use:
```
if *garbageCollectPtr {
    if sc.Verbosity > 2 {
        klog.Infof("---------- BEGIN GARBAGE COLLECTION: %s ----------\n",
			      *manifestPtr)
    }
...
```

From https://github.com/kubernetes-sigs/k8s-container-image-promoter/blob/27b1130f3e64859b874d04662551c28a50d332b2/cip.go#L224-#L226

Reply:
I was wondering if there is some klog best practice that we are missing here. Some research turned up the fact that klog is a fork of glog, which has support for verbosity levels https://godoc.org/github.com/golang/glog#V, but apparently I suppose klog doesn't support verbosity levels according to https://github.com/kubernetes/klog/issues/35.

So maybe for now instead of deleting all of the lines 72-125, maybe you can just replace the calls to `fmt.Println()` with the appropriate calls to klog. Wdyt?

Reply:
Yeah that's probably the most efficient way, I couldn't think of a way to support verbosity levels without an if block for every call. I'll update my branch.

I wonder what the reasoning was for _removing_ verbosity levels support, hopefully that gets brought back.

Reply:
Correction, I do still see it here - https://github.com/kubernetes/klog/blob/master/klog.go#L1107-#L1134 - so I'll see if this is viable.

Reply:
SG. I would like to add that according to https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md the klog.Infof() has multiple logging levels (e.g., `klog.V(1)`). I think we can just use those, probably.